RxBlocking/BlockingObservable+Operators.swift

```diff
@@ -10,14 +10,20 @@
     import RxSwift
 #endif
 
```

Comment:
I've tweaked the `UnitTests.md` and added an example of the successful case, not just the error case.

Although if you mean inline docs like @freak4pc mentioned, I can add that.